### ISSUE TITLE
Errors should not return http 200

### ISSUE THREAD

**Issue by ddewaele** [marksull/udooneorest#3]
When executing an "invalid" call that returns an error, the service should not return a HTTP 200 error, but rather some kind of 500 internal server error.

```
curl -v http://localhost:8085/udooneorest/gpio/102/value

*   Trying ::1...
* connect to ::1 port 8085 failed: Connection refused
*   Trying fe80::1...
* connect to fe80::1 port 8085 failed: Connection refused
*   Trying 127.0.0.1...
* Connected to localhost (127.0.0.1) port 8085 (#0)
> GET /udooneorest/gpio/102/value HTTP/1.1
> Host: localhost:8085
> User-Agent: curl/7.43.0
> Accept: */*
> 
< HTTP/1.1 200 OK 
< Content-Type: text/html;charset=utf-8
< Content-Length: 196
< X-Xss-Protection: 1; mode=block
< X-Content-Type-Options: nosniff
< X-Frame-Options: SAMEORIGIN
< Server: WEBrick/1.3.1 (Ruby/2.2.1/2015-02-26)
< Date: Sat, 19 Dec 2015 15:19:24 GMT
< Connection: close
< 
* Closing connection 0
{'status' : 'failed', 'message' : 'GPIO value of '102' is invalid. Should be one of 106,107,116,119,124,124,127,14,15,16,17,172,173,174,175,176,177,18,180,181,182,19,20,202,203,21,22,25,4,5,6,7.'}
```

**Comment by marksull** [comment]
Sorted, thanks. Returning 400 - Bad request. Included in 1.1.0.

**Comment by ddewaele** [comment]
Thx a lot ... spent 1 hour on stackoverflow / sinatra docs but couldn't get it to work. Zero ruby experience....
